app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,6 @@
+class FoodsController < ApplicationController
+  def search
+    @hits = FoodDataCentralService.ingredient_search(params[:q])[:totalHits]
+    @foods = FoodsFacade.foods_by_ingredient(params[:q])
+  end
```

Comment:
You're currently making two api calls to the same exact endpoint. Can you find a way to make one call to get ALL the data you need at once?